lms/djangoapps/teams/tests/test_views.py

```diff
@@ -186,7 +202,7 @@ def make_call(self, url, expected_status=200, method='get', data=None, content_t
 
         If a user is specified in kwargs, that user is first logged in.
         """
-        user = kwargs.pop('user', 'student_enrolled')
+        user = kwargs.pop('user', 'student_enrolled_not_on_team')
```

Comment:
@dianakhuang this is a good catch. Have we covered this elsewhere to make it clear to the user that they can't create a new team if they are already on one? I don't think we have. If not, I'll make a small story to address.